src/UI/Component/Listing/Factory.php

```diff
@@ -46,4 +46,28 @@ public function ordered(array $items);
 	 */
 	public function descriptive(array $items);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      A workflow presents a list of steps that the user should tackle
+	 *      in a defined order.
```

Comment:
Could you please drop the requirement for strict ordering. We use this kind of list and the steps are not required to be completed in a strict order but could be completed in a different order. 
![checklist](https://user-images.githubusercontent.com/24959113/41534560-442e309c-72ff-11e8-9890-44c1a9468050.PNG)
However what is really important for this list is the "diagnostic by-line" indicating what is wrong / what specifically still needs completing.

Reply:
![privacy checklist_1](https://user-images.githubusercontent.com/24959113/42268276-65fc6422-7f7b-11e8-9dc3-229ad3ccebac.PNG)
![checklist for course](https://user-images.githubusercontent.com/24959113/42268278-661a65d0-7f7b-11e8-8641-005fa108bc7a.PNG)
![privacy checklist_2](https://user-images.githubusercontent.com/24959113/42268279-66371eaa-7f7b-11e8-9b9e-0485b034bcee.PNG)
Oliver Samoila has provided mock-ups for the Checklists